### ISSUE TITLE
Error message about 'params' when importing a .json/.ink

### ISSUE THREAD

**Issue by paulloz** [paulloz/godot-ink#22]
This error message shows up every time a `.json`/`.ink` file is imported through the addon.
```
ERROR: Cannot erase nonexistent section "params".
   At: core/io/config_file.cpp:124
```

It seems to be unharmful.

**Comment by paulloz** [comment]
This is caused by godotengine/godot#38864.

**Comment by paulloz** [comment]
My PR fixing this in the engine has been merged: godotengine/godot#38876.  
Keeping this issue open for reference as long as the patch is not included in a stable release of the engine.
